dev_setup.sh

```diff
@@ -74,6 +74,10 @@ for var in "$@" ; do
     if [[ ${var} == "-fm" ]] ; then
         opt_forcemimicbuild=true
     fi
+    fi [[ ${var} != "-n" ]] || [[ ${var} != "--no-error" ]] ; then
+    	# exit on any error
+	set -Ee
```

Comment:
This is mainly me thinking loud: Would it make sense to have `set -Ee` at the top of the file and then on --no-error `set +Ee` to turn off the errors.

This might be good to easily detect errors in the startup or argument parsing code.